app/src/main/res/values/strings.xml

```diff
@@ -386,7 +386,7 @@
     <string name="pref_header_notifications">Notifications</string>
 
     <string name="notification_custom_rules">Custom rules</string>
-    <string name="notification_tip_no_custom_rules">There are no custom rules. You can add one with the plus button in the bottom right corner.</string>
+    <string name="notification_tip_no_custom_rules">There are no custom rules. You can add one with the \"+\" button in the bottom right corner.</string>
```

Comment:
To be honest, I'd have to see how does "+" look in the app internally/see how other apps do it. I'm unsure whether I approve of this or not until then.